### PR TITLE
OPENEUROPA-2936: PHP 7.3. - Update Code Review component.

### PR DESCRIPTION
**Pull request by sergepavle**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -4,14 +4,14 @@ workspace:
 
 services:
   web:
-    image: fpfis/httpd-php-ci:${PHP_VERSION=7.1}
+    image: fpfis/httpd-php-ci:${PHP_VERSION=7.2}
```
Comment (brummbar) [inline .drone.yml]: The variable is always defined, let's remove the default value.

Comment (voidtek) [reply .drone.yml]: The default value is used if you want to test your pipeline with drone-cli in your local machine.
Drone-cli doesn't use the matrix.

Comment (sergepavle) [reply .drone.yml]: @brummbar this is common practice in all other components.
If you still think and disagree with input from @voidtek I could remove but let's leave this for at least for consistance.

Comment (brummbar) [reply .drone.yml]: We talked with @ademarco and decided to drop it.

Comment (sergepavle) [reply .drone.yml]: Done

**Pull request merged**